src/platform/forms/save-in-progress/ApplicationStatus.jsx

```diff
@@ -137,7 +137,7 @@ export class ApplicationStatus extends React.Component {
           .format('MMMM D, YYYY [at] h:mm a');
 
         return (
-          <div className="usa-alert usa-alert-info background-color-only sip-application-status">
+          <div className="usa-alert usa-alert-info background-color-only sip-application-status vads-u-margin-bottom--2 vads-u-margin-top--0">
```

Comment:
Does the `sip-application-status` class still need to be here? It looks like you deleted it above